src/pages/List/index.js

```diff
@@ -1,15 +1,42 @@
 import React from 'react';
-import ItemsList from '../../components/items-list';
+import { useHistory } from 'react-router-dom';
 import Title from '../../components/title';
 import ContentContainer from '../../components/content-container';
 import Navigation from '../../components/routing/Navigation';
+import Button from '../../components/button';
+import { useList } from '../../hooks/useList';
 
-export const ListPage = () => (
-  <>
-    <Title>List</Title>
-    <ContentContainer>
-      <ItemsList />
-    </ContentContainer>
-    <Navigation />
-  </>
-);
+export const ListPage = () => {
+  const { list, loading, error } = useList();
+  const { push } = useHistory();
+
+  const showProducts = () => {
+    if (list.length === 0) {
+      return <p>You don't have any product in your list </p>;
+    }
+
+    return (
+      <ul>
+        {list.map(({ date, productName }) => (
+          <li key={date}>{productName}</li>
+        ))}
+      </ul>
+    );
+  };
+
+  return (
+    <>
+      <Title>List</Title>
+      <ContentContainer>
+        {loading ? <p>Loading...</p> : null}
+        {!loading && error ? (
+          <Button onClick={() => push('/')}>
+            There was an error, try again
+          </Button>
```

Comment:
On mobile devices, this button might be too wide to fit the screen :thinking: 
What if you sent a message above the button with the `There was an error.` and the button displays the `Try again` ?.

Btw, If I misswrote the token like this (see I missing the R in _sorb_)
![image](https://user-images.githubusercontent.com/37032132/138298384-d27b403d-8d69-4cf4-8d2c-93d022124c65.png)
Cases like these, should be better to tell the user we didnt find this token before sending them to the products list.

Reply:
Yan, good button suggestion